cw_bitcoin/lib/electrum.dart

```diff
@@ -235,21 +235,23 @@ class ElectrumClient {
         return [];
       });
 
-  Future<List<Map<String, dynamic>>> getListUnspent(String scriptHash) =>
-      call(method: 'blockchain.scripthash.listunspent', params: [scriptHash])
-          .then((dynamic result) {
-        if (result is List) {
-          return result.map((dynamic val) {
-            if (val is Map<String, dynamic>) {
-              return val;
-            }
+  Future<List<Map<String, dynamic>>> getListUnspent(String scriptHash) async {
+    final result = await call(method: 'blockchain.scripthash.listunspent', params: [scriptHash]);
 
-            return <String, dynamic>{};
-          }).toList();
+    if (result == null) throw Exception('Failed to get listunspent');
```

Comment:
this will have side effects
ex:
1. it's called in a `Future.wait`, so, throwing an exception would cause the whole Future.wait to fail
2. the second place it's called, has a try and catch, but it will make it not continue the flow
3. and also would make it report that error to the user through the send error report popup, thus giving the support more unnecessary tickets for connection issues